packages/react-composites/src/composites/ChatComposite/ChatScreen.tsx

```diff
@@ -134,13 +134,18 @@ export const ChatScreen = (props: ChatScreenProps): JSX.Element => {
     useImageUpload();
   /* @conditional-compile-remove(rich-text-editor-image-upload) */
   const [textOnlyChat, setTextOnlyChat] = useState(false);
+  /* @conditional-compile-remove(rich-text-editor-image-upload) */
+  const [isACSChat, setACSChat] = useState(false);
 
   /* @conditional-compile-remove(rich-text-editor-image-upload) */
   useEffect(() => {
     const updateChatState = (newState: ChatAdapterState): void => {
       setTextOnlyChat(newState.thread.properties?.messagingPolicy?.textOnlyChat === true);
+      setACSChat(newState.thread.properties?.createdBy?.kind !== 'microsoftTeamsUser');
```

Comment:
Should define 'microsoftTeamsUser' somewhere common?

Reply:
It looks like we use strings literals everywhere for participant types (like 'phoneNumber'). I'll create a func to check the type though

Reply:
Updated to use a func from common package